### PR TITLE
Hazard sfincs into fiat integrator

### PR DESCRIPTION
**Pull request by Mares2022**
Modifications in hazard code and fiat from writing in toml file. The test codes are connected to the pdrive. A single catalog contains all the datasets references.

### CODE REVIEW

File: fiat.py
Comment (frederique-hub) [inline fiat.py:1]: Why is this file here? Can you remove this if it is not being used?

File: hydromt_fiat/workflows/hazard.py [hunk 1]
```diff
@@ -308,71 +327,97 @@ def fstrip(x):
...
+            
+            
 
             model_fiat.set_maps(da, da_name)
             # post = f"(rp {da_rp})" if rp is not None and risk_output else ""
             post = f"(rp {da_rp})" if risk_output else ""
             model_fiat.logger.info(f"Added {hazard_type} hazard map: {da_name} {post}")
 
-            # new_da = xr.concat([new_da, da], dim='rp')
+            # new_da = xr.concat([new_da, da], dim='rp')
```
Comment (frederique-hub) [inline hydromt_fiat/workflows/hazard.py:338]: Can you delete the lines from this whole file that are commented out and that we are for sure not using at a later stage?

File: hydromt_fiat/workflows/hazard.py [hunk 2]
```diff
@@ -308,71 +327,97 @@ def fstrip(x):
...
             new_da = maps_0.to_dataset(name='RISK')
             new_da.attrs = {  "returnperiod": list(list_rp),
                             "type":self.map_type_lst,
                             'name':list_names,
                             "Analysis": "Risk"}  
 
-        model_fiat.hazard = new_da
-        model_fiat.set_maps(model_fiat.hazard, 'HydroMT_Fiat_hazard')
+            model_fiat.hazard = new_da
+            model_fiat.set_maps(model_fiat.hazard, 'HydroMT_Fiat_hazard')
```
Comment (frederique-hub) [inline hydromt_fiat/workflows/hazard.py:362]: Where and how is this HydroMT_Fiat_hazard name used?

File: tests/test_SVI.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 import pytest
 
 
-EXAMPLEDIR = Path().absolute() / "local_test_database"
+EXAMPLEDIR = Path("P:/11207949-dhs-phaseii-floodadapt/Model-builder/Delft-FIAT/local_test_database")
```
Comment (frederique-hub) [inline tests/test_SVI.py:7]: For now it is okay to point to the P-Drive but we should find a better solution for this soon.

**Review by frederique-hub — CHANGES_REQUESTED** [review]
Seems good, I requested a few small changes. After that we can merge the branches

File: hydromt_fiat/workflows/hazard.py (hunk 2 above)
Comment (Mares2022) [reply hydromt_fiat/workflows/hazard.py:362]: This part is also not needed, I made this mainly to have an attribute .hazard in the model that would trigger the writing component (if model.hazard then write in the toml file)

The line can be simplified as: model_fiat.set_maps(new_da, 'key of the map'). The key of the map can also be "risk_maps" we can define it when having a look into the outputs of the model builder.

File: fiat.py
Comment (Mares2022) [reply fiat.py:1]: Yes, you are right it is not used. I have just erased
